IdentityCore/src/webview/MSIDWebviewAuthorization.m

```diff
@@ -26,34 +26,185 @@
+                                          context:(id<MSIDRequestContext>)context
+                                completionHandler:(MSIDWebviewAuthCompletionHandler)completionHandler
+{
+    //    NSString *state = [factory generateStateValue];
+    //    NSURL *startURL = [factory startURLFromConfiguration:configuration requestState:state];
+    //    (void)startURL;
+    //    (void)state;
+    //
+    //    MSIDOAuth2EmbeddedWebviewController *embeddedWebviewController = [[MSIDOAuth2EmbeddedWebviewController alloc] init];
+    //    [self startWebviewAuth:embeddedWebviewController
+    //                   factory:factory
+    //              requestState:state
+    //                   context:context
+    //         completionHandler:completionHandler];
+}
+
++ (void)startEmbeddedWebviewWebviewAuthWithConfiguration:(MSIDWebviewConfiguration *)configuration
+                                           oauth2Factory:(MSIDOauth2Factory *)oauth2Factory
+                                             verifyState:(BOOL)verifyState
+                                                 webview:(WKWebView *)webview
+                                                 context:(id<MSIDRequestContext>)context
+                                       completionHandler:(MSIDWebviewAuthCompletionHandler)completionHandler
+{
+    //    NSString *state = [factory generateStateValue];
+    //    NSURL *startURL = [factory startURLFromConfiguration:configuration requestState:state];
+    //    (void)startURL;
+    //    (void)state;
+    //
+    //    MSIDOAuth2EmbeddedWebviewController *embeddedWebviewController = [[MSIDOAuth2EmbeddedWebviewController alloc] init];
+    //    [self startWebviewAuth:embeddedWebviewController
+    //                   factory:factory
+    //              requestState:state
+    //                   context:context
+    //         completionHandler:completionHandler];
+}
+
+#if TARGET_OS_IPHONE
++ (void)startSystemWebviewWebviewAuthWithConfiguration:(MSIDWebviewConfiguration *)configuration
+                                         oauth2Factory:(MSIDOauth2Factory *)oauth2Factory
+                                           verifyState:(BOOL)verifyState
+                                               context:(id<MSIDRequestContext>)context
+                                     completionHandler:(MSIDWebviewAuthCompletionHandler)completionHandler
+{
+    MSIDWebviewFactory *webviewFactory = [oauth2Factory webviewFactory];
+    MSIDWebviewSession *session = [webviewFactory systemWebviewSessionFromConfiguration:configuration verifyState:verifyState context:context];
+    
+    [self startSession:session verifyState:verifyState context:context completionHandler:completionHandler];
+}
+#endif
+
++ (void)startSession:(MSIDWebviewSession *)session
+         verifyState:(BOOL)verifyState 
+             context:(id<MSIDRequestContext>)context
+   completionHandler:(MSIDWebviewAuthCompletionHandler)completionHandler
+{
```

Comment:
parameter `verifyState` is no longer used in this function anymore :)

Reply:
Let's see if there is a need for anyone to override this value from default.